### PR TITLE
[2.6.x] Move PR validation to AzDO

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        _BuildConfig: Debug
+      Release:
+        _BuildConfig: Release
+  steps:
+  - checkout: self
+    clean: true
+  - script: .\eng\common\CIBuild.cmd -configuration $(_BuildConfig) -prepareMachine
+  - task: CopyFiles@2
+    inputs:
+      contents: 'artifacts/**
```
Comment (chcosta) [inline azure-pipelines.yml]: missing closing `'`

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+trigger:
+- 2.6.x
+- 2.9.x
```
Comment (mavasani) [inline azure-pipelines.yml]: Note that our current active branches are master, 2.6.x and 3.x
We are no longer using 2.9.x

Tagging @sharwell

File: azure-pipelines.yml (hunk 1 above)
Comment (mmitche) [reply azure-pipelines.yml]: agh i knew it was something stupid

File: azure-pipelines.yml (hunk 2 above)
Comment (mmitche) [reply azure-pipelines.yml]: This is primarily about getting off of Jenkins.  So 2.9.x (tracked in Jenkins) wont' ever be touched again?

Comment (mavasani) [reply azure-pipelines.yml]: Yes, we don't intend to use that branch again.

**Pull request merged**